hooks/command

```diff
@@ -126,22 +126,28 @@ echo "Timeout: ${timeout}s"
 
 echo "--- :kubernetes: Running image: ${BUILDKITE_PLUGIN_K8S_IMAGE}"
 
+error_file=$(mktemp)
 counter="$timeout"
 while true
 do
+  echo -n > "$error_file"
   set +e
-  pod_name=$(kubectl get pod -l "job-name=$job_name" --output=jsonpath="{.items[*].metadata.name}")
+  pod_name=$(kubectl get pod -l "job-name=$job_name" --output=jsonpath="{.items[*].metadata.name}" 2>"$error_file")
```

Comment:
we can avoid creating a file in order to check the output e.g:
```sh
pod_name=$(kubectl get pod -l "job-name=$job_name" --output=jsonpath="{.items[*].metadata.name}" 2>&1)
if [[ "$?" -ne 0 ]] && [[ $pod_name = *"NotFound"* ]]; then
  # We know that the pod doesn't exist so we can check the job status
  jobstatus=$(kubectl get job "$job_name" -o 'jsonpath={.status.conditions[].type}')
  ...
fi
# If the pod failed for another reason,then we want to sleep and retry
# so the if [[ -n "$pod_name" ]] condition can be replaced with if [[ "$exit_code -eq 0 ]]
# since $pod_name now can contain stderr
```

Reply:
Yea I was just trying to avoid getting other output from stderr in the pod name if there was some sort of warning, and therefore this command was successful

Reply:
updated to do this